### PR TITLE
Group a keo

### PR DESCRIPTION
**Pull request by K-Magano**
Created Instructions component and completed #8

### CODE REVIEW

File: helpers/mongoDB-utils.js
Comment (Renzo-codespace) [inline helpers/mongoDB-utils.js:1]: Best practice would be get the entire connection string from the .env file. 
Having it partly hardcoded like this means this program can only connect to that db, but if it comes entirely from the .env,  the db it connects can be easily changed.

End each try block where you use the client with finally {client.close()}

File: next.config.js
Comment (Renzo-codespace) [inline next.config.js:1]: Usernames, passwords and other database details should NEVER be committed.

**Review by Cbazil — DISMISSED** [review]

**Review by Renzo-codespace — CHANGES_REQUESTED** [review]
Please remove the all sensitive data from the code base.

**Review by Renzo-codespace — DISMISSED** [review]
Issues I raised earlier can be addressed when we start implementing best practices.

**Review by Renzo-codespace — APPROVED** [review]
Code shouldn't be commented out, rather deleted.

**Review by Cbazil — APPROVED** [review]

**Pull request merged**